src/lib.rs

```diff
@@ -1023,8 +990,8 @@ mod tests {
         assert_eq!(payload, expected_payload);
     }
 
-    #[test]
-    fn test_response() {
+    #[tokio::test]
+    async fn test_response() {
```

Comment:
this test is now async tokio, and it passes, but probably i am doing something wrong.